### ISSUE TITLE
[BUG] VJ code not detected, also I write a script for the DLsite metadata extension

### ISSUE THREAD

**Issue by RoyalShooter** [erri120/Playnite.Extensions#97]
Hi there, Thank you for making the DLsite metadata extensions, is solve my biggest problems with Games library problems. I have written a little script to let your code run better by preparing the input data to DL code. Hope you don't mind I mention you in my project.

[https://github.com/RoyalShooter/Playnite-DLsite-Metadata-Rename-to-DL-ID](https://github.com/RoyalShooter/Playnite-DLsite-Metadata-Rename-to-DL-ID)

I would also like to report an issue I found when I am writing the script, I found the DLsite extension works when the name field is in RJ code (RJ123456) but not with VJ code (VJ123456), but if I put a full link in the name file it works again even it links to a VJ code game, not sure if this is a bug or not.

Anyway, thanks again for writing a great extension.

**Comment by erri120** [comment]
Do you have some example `VJ` ids? Also I think the search by name is currently broken (#95)

**Comment by RoyalShooter** [comment]
Yes I can confirm search by name is broken, that is the reason why I write this script in the first place to help the extension get the ID.

**Comment by RoyalShooter** [comment]
> Do you have some example `VJ` ids? Also I think the search by name is currently broken (#95)

For example, on VJ014028 if I put just this"VJ014028" ID into name field it won't download the metadata, but if I put the full like 
"https://www.dlsite .com/pro/work/=/product_id/ $GameID .HTML" (I don't want to get ban so I don't put a link here)
this link in the name field will work for the metadata download and the extension works.

Compared to RJ code works by just the ID in the name field.

**Comment by erri120** [comment]
Hmm, looks like VJ is mostly under `/pro` and `/soft`
```raw
/maniax/.../RJ
/pro/.../VJ
/soft/.../VJ
```

**Comment by RoyalShooter** [comment]
> Hmm, looks like VJ is mostly under `/pro` and `/soft`
> 
> ```
> /maniax/.../RJ
> /pro/.../VJ
> /soft/.../VJ
> ```

yes, but even if you search it under /maniax/ site it give out result for /pro/ site. In my script I only search the maniax site and that is enough for me to also get results for VJ code.

would you mind me to provide my script to #95 as a temporary solution?
